app/models/apartment.rb

```diff
@@ -0,0 +1,13 @@
+class Apartment < ApplicationRecord
+    belongs_to :user
+    validates :street, presence: true
+    validates :unit, presence: true
+    validates :city, presence: true
+    validates :state, presence: true
+    validates :square_footage, presence: true
+    validates :price, presence: true
+    validates :bedrooms, presence: true
+    validates :bathrooms, presence: true
+    validates :pets, presence: true
+    validates :image, presence: true
+end
```

Comment:
Certainly works, I recommend to refactor to all the attributes being on one line with the presence validation helper.